src/caffe/layers/python_layer.cpp

```diff
@@ -0,0 +1,56 @@
+#ifdef WITH_PYTHON_LAYER
+
+#include "caffe/python_layer.hpp"
+
+#include <boost/thread.hpp>
+#include <vector>
+
+namespace caffe {
+
+class aquire_py_GIL {
```

Comment:
Should we CamelCase this for style consistency? (Or is this intentionally made to look like a boost-style underscore_case lock or some such thing?)